src/com/github/davidtcalabrese/CalculatorInput.java

```diff
@@ -1,5 +1,6 @@
 package com.github.davidtcalabrese;
-/** this class serves as a blueprint for CalculatorInput objects
+/**
+ * This class serves as a blueprint for CalculatorInput objects
```

Comment:
remember to focus on one thing at a time. when you format a comment while in a issue branch that shows up as a diff but doesn't have anything to do with the issue